scripts/gulp/create-bundle.js

```diff
@@ -10,6 +10,7 @@ const babelify = require('babelify');
 const browserify = require('browserify');
 const coffeeify = require('coffeeify');
 const exorcist = require('exorcist');
+const envify = require('loose-envify/custom');
```

Comment:
Just an observation that we seem to have a lot of dependencies for bundling here.

Reply:
Yeah, it is part of the philosophy of Browserify to have minimal functionality in the core.  When I lasted evaluated our options it was still the best fit for us. That was a couple of years ago though. For other projects we might want to consider a more “batteries included” option, like say ParcelJS or Webpack.